bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1188 @@
+			<channel id="prodCode" typeId="prodCode"/>
+			<channel id="resetFlag" typeId="resetFlag"/>
+			<channel id="factorySettingWzu" typeId="factorySettingWzu"/>
+			<channel id="factoryReset" typeId="factoryReset"/>
+			<!--  System Settings -->
+			<channel id="language" typeId="language"/>
+			<channel id="dateFormat" typeId="dateFormat"/>
+			<channel id="date" typeId="date"/>
+			<channel id="time" typeId="time"/>
+			<channel id="timeZoneDifferenceToGmt" typeId="timeZoneDifferenceToGmt"/>
+			<channel id="summerWinter" typeId="summerWinter"/>
+			<!--  Software / Firmware -->
+			<channel id="softwareVersionBasis" typeId="softwareVersionBasis"/>
+			<channel id="autoSwUpdate" typeId="autoSwUpdate"/>
+			<channel id="accessHeliosPortal" typeId="accessHeliosPortal"/>
+			<channel id="globalManualWebUpdate" typeId="globalManualWebUpdate"/>
+			<channel id="portalGlobalsErrorForWeb" typeId="portalGlobalsErrorForWeb"/>
+			<channel id="logout" typeId="logout"/>
+		</channels>
+	</channel-group-type>
+	
+	<channel-group-type id="operation">
+		<label>Operation</label>
+		<description>Control of operating mode</description>
+		<channels>
+			<!--  Party Mode -->
+			<channel id="partyModeDuration" typeId="partyModeDuration"/>
+			<channel id="partyModeFanStage" typeId="partyModeFanStage"/>
+			<channel id="partyModeRemainingTime" typeId="partyModeRemainingTime"/>
+			<channel id="partyModeStatus" typeId="partyModeStatus"/>
+			<!--  Standby Mode -->
+			<channel id="standbyModeDuration" typeId="standbyModeDuration"/>
+			<channel id="standbyModeFanStage" typeId="standbyModeFanStage"/>
+			<channel id="standbyModeRemainingTime" typeId="standbyModeRemainingTime"/>
+			<channel id="standbyModeStatus" typeId="standbyModeStatus"/>
+			<!--  Holiday Programme -->
+			<channel id="holidayProgramme" typeId="holidayProgramme"/>
+			<channel id="holidayProgrammeFanStage" typeId="holidayProgrammeFanStage"/>
+			<channel id="holidayProgrammeStart" typeId="holidayProgrammeStart"/>
+			<channel id="holidayProgrammeEnd" typeId="holidayProgrammeEnd"/>
+			<channel id="holidayProgrammeInterval" typeId="holidayProgrammeInterval"/>
+			<channel id="holidayProgrammeActivationTime" typeId="holidayProgrammeActivationTime"/>
+			<!--  Operating Mode / Fan Stage-->
+			<channel id="operatingMode" typeId="operatingMode"/>
+			<channel id="fanStage" typeId="fanStage"/>
+			<channel id="percentageFanStage" typeId="percentageFanStage"/>
+			<channel id="extractAirFanStage" typeId="fanStage"/>
+			<channel id="supplyAirFanStage" typeId="fanStage"/>
+		</channels>
+	</channel-group-type>
+	
+	<channel-group-type id="unitConfig">
+		<label>Unit Config</label>
+		<description>Configuration parameters of the ventilation unit</description>
+		<channels>
+			<!--  Fan Stage Configuration -->
+			<channel id="minFanStage" typeId="minFanStage"/>
+			<channel id="assignmentFanStages" typeId="assignmentFanStages"/>
+			<channel id="fanStageStepped0to2v" typeId="fanStageStepped1"/>

```

Comment:
You could specify the label here, to avoid specifying an individual type for each Channel.